sdk/messaging/azeventhubs/internal/eh/stress/templates/deploy-job.yaml

```diff
@@ -21,30 +22,36 @@ spec:
       - batch
       - "-rounds"
       - "100"
+      - "-verbose"
```

Comment:
Can -verbose be a flag that gets set at a higher level and flows everywhere?

Reply:
It could be, but with these args I like it to be as flat and DRY as possible, since it's annoying to have to trace inheritance/fallthroughts and such in templates.